### PR TITLE
[SW-445, SW-576] remove default browser input validation on amounts

### PR DESCRIPTION
**Pull request by ifaouibadi**

### CODE REVIEW

File: src/popup/components/InputField.vue [hunk 1]
```diff
@@ -59,7 +59,6 @@
             autocomplete="off"
             step="any"
             data-cy="input"
-            :type="type"
```
Comment (peronczyk) [inline src/popup/components/InputField.vue:62]: Are you sure? The input should have the type property. Here we are using the component's property `type`.

Comment (peronczyk) [inline src/popup/components/InputField.vue:62]: Are you sure? The input should have the type property. Here we are using the component's property type.

File: src/popup/components/TransferSendForm.vue [hunk 2]
```diff
@@ -85,7 +85,7 @@
     </div>
 
     <InputAmount
-      v-model="formModel.amount"
+      v-model.number="formModel.amount"
```
Comment (CedrikNikita) [inline src/popup/components/TransferSendForm.vue:88]: Even if you want to stick to this approach there are ~10 places where `InputAmount` is used, and where this line should be used in this case.

File: src/popup/components/InputField.vue (hunk 1 above)
Comment (ifaouibadi) [reply src/popup/components/InputField.vue]: Reverted it as `v-model.number` resolves both issues reported on SW-576

File: src/popup/components/TransferSendForm.vue (hunk 2 above)
Comment (ifaouibadi) [reply src/popup/components/TransferSendForm.vue:88]: updated for other components too

File: src/popup/components/InviteItem.vue [hunk 3]
```diff
@@ -38,7 +38,7 @@
     </div>
     <template v-else>
       <InputAmount
-        v-model="topUpAmount"
+        v-model.number="topUpAmount"
```
Comment (CedrikNikita) [inline src/popup/components/InviteItem.vue:41]: The number approach will break the long numbers logic we have, and we will lose precision.

**Review by CedrikNikita — CHANGES_REQUESTED** [review]

**Review by CedrikNikita — APPROVED** [review]

**Review by peronczyk — APPROVED** [review]

**Pull request merged**